TechnicoServer/TechnicoServer/Models/PropertyItem.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace TechnicoDomain.Models;
+
+public class PropertyItem
+{
+    public long Id { get; set; }
+    public string Name { get; set; } = string.Empty;
```

Comment:
Reused naming here, you use the "Name" variable for both models PropertyItem and Owner, I would suggest using another name.